docs/modeling/fitting.rst

```diff
@@ -123,3 +93,89 @@ bounds internally.
     ['fixed', 'tied', 'bounds']
     >>> fitting.SLSQPLSQFitter.supported_constraints
     ['bounds', 'eqcons', 'ineqcons', 'fixed', 'tied']
+
+
+Fitting examples
+================
+
+Simple 2-D model fitting
+------------------------
+
+Similarly to the :ref:`1-D example <1D-fitting>`, we can create a simulated 2-D data dataset, and
+fit a polynomial model to it. This could be used for example to fit the
+background in an image.
+
+.. plot::
+   :include-source:
+
+    import warnings
+    import numpy as np
+    from astropy.modeling import models, fitting
+
+    # Generate fake data
+    np.random.seed(0)
+    y, x = np.mgrid[:128, :128]
+    z = 2. * x ** 2 - 0.5 * x ** 2 + 1.5 * x * y - 1.
+    z += np.random.normal(0., 0.1, z.shape) * 50000.
+
+    # Fit the data using astropy.modeling
+    p_init = models.Polynomial2D(degree=2)
+    fit_p = fitting.LevMarLSQFitter()
+
+    with warnings.catch_warnings():
+        # Ignore model linearity warning from the fitter
+        warnings.simplefilter('ignore')
+        p = fit_p(p_init, x, y, z)
+
+    # Plot the data with the best-fit model
+    plt.figure(figsize=(8,2.5))
+    plt.subplot(1,3,1)
+    plt.imshow(z, origin='lower', interpolation='nearest', vmin=-1e4, vmax=5e4)
+    plt.title("Data")
+    plt.subplot(1,3,2)
+    plt.imshow(p(x, y), origin='lower', interpolation='nearest', vmin=-1e4,
+               vmax=5e4)
+    plt.title("Model")
+    plt.subplot(1,3,3)
+    plt.imshow(z - p(x, y), origin='lower', interpolation='nearest', vmin=-1e4,
+               vmax=5e4)
+    plt.title("Residual")
+
+Fitting to multiple datasets
+----------------------------
+
+The package also allows fitting a polynomial model to multiple data sets
+simultaneously::
+
+      >>> from astropy.modeling import models, fitting
+      >>> import numpy as np
+      >>> p1 = models.Polynomial1D(3)
+      >>> p1.c0 = 1
+      >>> p1.c1 = 2
```

Comment:
You can also pass the parameters in to the constructor: `Polynomial1D(3, c0=1, c1=2)`.  Probably fine since they're just initial guesses.  Unless you intended to show this usage (maybe it's easier to understand?)  Up to you.